### DISCUSSION TITLE
How to see GitHub Projects

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#234] [General]
So if you haven't published your site or others haven't published theirs, there is a way to see their webpage. Here are a few steps:

1. Open their github page
2. Go to their index.html
3. Paste "https://htmlpreview.github.io/?" before the webpage
4. Click enter and see their new webpage

**Comment by CJ3315** [comment]
Awesome thank you so much! I was just wondering about this!
